pkg/workloads/console/integration/integration_test.go

```diff
@@ -216,68 +217,6 @@ var _ = Describe("Console", func() {
 				"the console expiry time should be after now()",
 			)
 		})
-
-		It("Deletes the job after the expiry time", func() {
-			By("Retrieving latest console object")
-			updatedCsl := &workloadsv1alpha1.Console{}
-			identifier, _ := client.ObjectKeyFromObject(csl)
-			err := mgr.GetClient().Get(context.TODO(), identifier, updatedCsl)
-			Expect(err).NotTo(HaveOccurred(), "failed to retrieve console")
-
-			By("Updating timeout to trigger console expiration")
-			updatedCsl.Spec.TimeoutSeconds = 0
-			err = mgr.GetClient().Update(context.TODO(), updatedCsl)
-			Expect(err).NotTo(HaveOccurred(), "failed to update console")
-
-			waitForSuccessfulReconcile(2)
-
-			job := &batchv1.Job{}
-			identifier, _ = client.ObjectKeyFromObject(csl)
-			err = mgr.GetClient().Get(context.TODO(), identifier, job)
-
-			statusError, _ := err.(*errors.StatusError)
-			Expect(err).To(HaveOccurred(), "the job should be deleted, but it still exists")
-			Expect(err).To(BeAssignableToTypeOf(statusError), "error should be an API error")
-			Expect(statusError.Status()).To(MatchFields(IgnoreExtras, Fields{"Code": BeEquivalentTo(404)}))
-		})
-
-		It("Does not recreate a console once it has expired", func() {
-			By("Expect that a job was created")
-			job := &batchv1.Job{}
-			identifier, _ := client.ObjectKeyFromObject(csl)
-			err := mgr.GetClient().Get(context.TODO(), identifier, job)
-
-			Expect(err).NotTo(HaveOccurred(), "failed to find associated Job for Console")
-
-			By("Retrieving latest console object")
-			updatedCsl := &workloadsv1alpha1.Console{}
-			identifier, _ = client.ObjectKeyFromObject(csl)
-			err = mgr.GetClient().Get(context.TODO(), identifier, updatedCsl)
-			Expect(err).NotTo(HaveOccurred(), "failed to retrieve console")
-
-			By("Updating timeout to trigger console expiration")
-			updatedCsl.Spec.TimeoutSeconds = 0
-			err = mgr.GetClient().Update(context.TODO(), updatedCsl)
-			Expect(err).NotTo(HaveOccurred(), "failed to update console")
-
-			waitForSuccessfulReconcile(2)
-
-			By("Updating console spec with a large timeout")
-			err = mgr.GetClient().Get(context.TODO(), identifier, updatedCsl)
-			Expect(err).NotTo(HaveOccurred(), "failed to retrieve console")
-			updatedCsl.Spec.TimeoutSeconds = 1000
-			err = mgr.GetClient().Update(context.TODO(), updatedCsl)
-			Expect(err).NotTo(HaveOccurred(), "failed to update console")
-
-			waitForSuccessfulReconcile(1)
-
-			job = &batchv1.Job{}
-			identifier, _ = client.ObjectKeyFromObject(csl)
-			err = mgr.GetClient().Get(context.TODO(), identifier, job)
-
-			Expect(err).To(HaveOccurred(), "the job should remain deleted, but it has been recreated")
-			Expect(errors.IsNotFound(err)).To(Equal(true), "a NotFound error was expected, but got a different error")
-		})
```

Comment:
The point of this test was to ensure that we only create a job once for every console. Since jobs will now live for the lifetime of the console, this can't happen (since we'd get a resource name clash). We could try testing that updating the timeout of a console after the job's pod has been killed doesn't recreate the pod, but that would be testing the job controller - and we can't do that anyway in this environment, because the job controller isn't running.